3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,69 @@
+//Flight booking
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (useFormalName == true && gender == "man") {
+    return `Lord ${firstname} ${surname}`;
+  } else if (useFormalName == true && gender == "woman") {
+    return `Miss ${firstname} ${surname}`;
+  } else {
+    return `${firstname} ${surname}`;
+  }
+}
+
+let fullname1 = getFullname("Sevda", "Naziri", true, "woman");
+let fullname2 = getFullname("John", "Wick", true, "man");
+
+console.log(fullname1);
+console.log(fullname2);
+
+//Event application
+let daysUntilEvent = 8;
+
+function getEventWeekday(days) {
+  var date = new Date();
+  date.setDate(date.getDate() + days);
+  return date;
+}
+console.log(getEventWeekday(daysUntilEvent));
+
+//Weather wear
+
+function whatShouldIWear(tempreture) {
+  if (tempreture >= 20) {
+    return `You should wear T-shirt and shorts}`;
+  } else if (tempreture < 20) {
+    return `You must wear warm cloths`;
+  }
+}
+console.log(whatShouldIWear(20));
+
```

Comment:
🎉 Good job taking on this coding challenge! 

🌞 Your function seems to be on the right track for suggesting clothing based on temperature. Nice start! 
💡 A minor suggestion: Instead of back-ticks (``), you might want to use regular quotation marks ("") around your clothing suggestions because you are not using any variables within the string.
🥶 just as an extra thought: what happens if the use does not provide any temperature to your function?